include/AutopinPlus/ControlStrategy.h

```diff
@@ -199,25 +193,34 @@ class ControlStrategy : public QObject {
 	QString name;
 
 	/*!
-	 * \brief Data structure for sorting tasks
-	 *
-	 * This struct is used in refreshTasks() and serves as comparison function for the
-	 * call of the sort function. Moreover, it stores a sort id for every. This id has
-	 * to be assigned before calling sort. The sorting is done with respect to the sort
-	 * ids (e. g. they could contain the creation time of a task) in ascending order. If
-	 * two sort ids are equal the tids are considered.
-	 */
-	struct tasksort {
-		sortmap sort_tasks;
-		bool operator()(int tida, int tidb) {
-			if (sort_tasks[tida] < sort_tasks[tidb])
-				return true;
-			else if (sort_tasks[tida] > sort_tasks[tidb])
-				return false;
-			else
-				return (tida < tidb);
-		}
-	};
+	 * \brief Stores the (configurable) interval between the regular
+	 * queries to the OS for the current list of threads. If intervall
+	 * is 0 or negative, polling is disabled;
+	 */
+	int interval = 100;
+
+  private:
+	/*!
+	 * \brief Stores the current pinning
+	 */
+	static Pinning pinning;
+
+	/*!
+	 * \brief Mutex for access to current_pinning
```

Comment:
Is this comment up to date? It looks like the mutex is guarding pinning not current_pinning.